e2e/e2e.md

```diff
@@ -0,0 +1,24 @@
+# End-to-End Testing with Playwright
+
+## Creating a test
+
+1. Go to the right e2e file and add a test with specific name.
+2. create the test with minimal overhead(less locators, less API calls etc.).
+
+## Run and Debug a test
+
+1. download playwright test for vscode extension
+2. You will see a small green play button near the test you created or want to run.
```

Comment:
I don't see a small green play button, maybe it is because we are using custom `test` in `fixture.js`?

Reply:
![image](https://github.com/os-scar/overlay/assets/109076747/64307f1f-cb45-40ea-96ba-8beaa16037a0)

Reply:
Not in my environment 🤷‍♂️